cmd/commands/runtime.go

```diff
@@ -585,6 +577,63 @@ func RunRuntimeInstall(ctx context.Context, opts *RuntimeInstallOptions) error {
 		return util.DecorateErrorWithDocsLink(fmt.Errorf("failed to create codefresh-cm: %w", err))
 	}
 
+	err = createRuntimeComponents(ctx, opts, rt)
+	if err != nil {
+		return err
+	}
+
+	err = createGitSources(ctx, opts)
+	if err != nil {
+		return err
+	}
+
+	timeoutErr := intervalCheckIsRuntimePersisted(ctx, opts.RuntimeName)
+	handleCliStep(reporter.InstallStepCompleteRuntimeInstallation, "Completing runtime installation", timeoutErr, true)
+	if timeoutErr != nil {
+		return util.DecorateErrorWithDocsLink(fmt.Errorf("failed to complete installation: %w", timeoutErr))
+	}
+
+	err = createGitIntegration(ctx, opts)
+	if err != nil {
+		return err
+	}
+
+	installationSuccessMsg := fmt.Sprintf("Runtime '%s' installed successfully", opts.RuntimeName)
+	skipIngressInfoMsg := `To complete the installation: 
+1. Configure your cluster's routing service with path to '/app-proxy' 
+2. Create and register Git integration using the commands:
+cf intg git add default --runtime <RUNTIME_NAME> --api-url <API_URL>
+cf intg git register default --runtime <RUNTIME_NAME>
+3. If you have chosen to install demo resources, configure a path to '/webhooks/push-git'`
```

Comment:
missing token here. try to run the whole flow and see you end up with full runtime installation